### ISSUE TITLE
Prettier: failed to parse buffer. Prettier fails on css files.

### ISSUE THREAD

**Issue by ciases** [prettier/vim-prettier#76]
Hello there!
For unknown reason prettier cannot style css files (there is no any problem with javascript by the way).
Example of css file:

```css
/* @format */

body {
        dislay:none ;
        opacity:   .75;
}
```
My environment:
- Ubuntu 16.04 x64
- `:PrettierCli` nothing
- `:PrettierCliPath` prettier
- `:PrettierCliVersion` 1.8.0
- `:PrettierVersion` 0.2.4
- `:version`
```                                                                                                                                                                       
NVIM v0.2.0                                                                                                                                                                                  
Build type: RelWithDebInfo                                                                                                                                                                   
Compilation: /usr/bin/x86_64-linux-gnu-gcc -g -O2 -fPIE -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -DDISABLE_LOG -Wdate-time -D_FORTIFY_SOURCE
=2 -Wconversion -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -g -DDISABLE_LOG -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wvla -fstack-protector-strong -fdiag
nostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -I/build/neovim-LpFVCC/neovim-0.2.0/build/config -I/build/neovim-LpFVCC/neovim-0.2.0/src -I/usr/include -I/usr/include -I/u
sr/include -I/usr/include -I/usr/include -I/usr/include -I/usr/include -I/build/neovim-LpFVCC/neovim-0.2.0/build/src/nvim/auto -I/build/neovim-LpFVCC/neovim-0.2.0/build/include             
Compiled by pkg-vim-maintainers@lists.alioth.debian.org                                                                                                                                      
                                                                                                                                                                                             
Optional features included (+) or not (-): +acl   +iconv    +jemalloc +tui                                                                                                                   
For differences from Vim, see :help vim-differences                                                                                                                                          
                                                                                                                                                                                             
   system vimrc file: "$VIM/sysinit.vim"                                                                                                                                                     
  fall-back for $VIM: "/usr/share/nvim"   
```

**Comment by adam-beck** [comment]
It does the same thing with `.json` files as well.

**Comment by mitermayer** [comment]
Hi @adam-beck,

Thanks for reporting this issue will look into it

**Comment by mitermayer** [comment]
Looks like something has changed on the new prettier `1.8` release. Will look into it and do a new release with the fixes as soon as possible

**Comment by mitermayer** [comment]
Managed to reproduce this issue on `prettier` `1.8.1` will investigate a bit more on this tomorrow to try to solve it.

This seems to be only happening only on `:PrettierAsync`, so in the mean time people can be unblocked by using the synchronous formatting `:Prettier`

**Comment by mitermayer** [comment]
So it turns out this was a regression introduced by https://github.com/prettier/vim-prettier/pull/74

*** 

At this stage reverted that PR and will revisit it again to make sure no regressions will be introduced

**Comment by chemzqm** [comment]
I'm ok with the revert, but I have no idea why the PR could cause this bug.

**Comment by vprasanth** [comment]
If you're using nvm like me, and forget to set a node version in the terminal you launch vim from -- this can happen as well. Just heads up :)

**Comment by stevematdavies** [comment]
I've almost given up on coc-prettier because of this, there seems to be no solution for this. Such a shame

**Comment by mitermayer** [comment]
@stevematdavies can you please open a new issue tracking this and I’m happy to revisit it

**Comment by stevematdavies** [comment]
Thanks for the offer, but due to time constraints, I have chosen an alternative. However, once I have the time, I will certainly open a new issue.
